swagger.json

```diff
@@ -1450,7 +1448,7 @@
         ],
         "responses": {
           "200": {
-            "description": "Flattened file tree with path relative to the user's root directory and some basic properties such as file size.",
+            "description": "Flattened file tree with path relative to the user's root directory and some basic properties such as the file size and the timestamp of the last modification. All properties except the name are optional.",
```

Comment:
We should clearly state which attributes can be supported and how they are formatted (e.g. timestamps as ISO8601?).